tidb-cloud/data-service-postman-integration.md

```diff
@@ -0,0 +1,87 @@
+    3. Copy the Data App URL in the TiDB Cloud dialog, and then past the URL to Postman for the import.
+
+5. After you paste the URL, Postman imports the Data App automatically as a new collection. The name of the connection is in the `TiDB Data Service - <Your App Name>` format.
+
+    In the connection, the deployed endpoints are grouped in the **Deployed** folder and the undeployed endpoints are grouped in the **Draft** folder.
+
+    > **Note:**
+    >
+    > Postman only imports endpoints that are ready for use. If an endpoint does not have target cluster, database, table, path, or SQL statements, the endpoint is skipped by the import.
+
+## Step 2. Configure your Data App API key in Postman
+
+Before running the imported Data App in Postman, you need to configure the API key for the Data App in Postman as follows:
+
+1. In the left navigation pane of Postman, click `TiDB Data Service - <Your App Name>` to open a tab for it on the right side.
+2. Under the `TiDB Data Service - <Your App Name>` tab, click the **Variables** tab.
+3. In the variable table, paste your public key and private key for your Data App to the **Current value** column.
+4. In the upper-right corner of the `TiDB Data Service - <Your App Name>` tab, click **Save**.
+
+## Step 3. Run Data App in Postman
+
+1. In the left navigation pane of Postman, expand the **Deployed** or **Draft** folder, and then click your endpoint name to open a tab for it on the right side.
+2. Under the `<Your Endpoint Name>` tab, you can call your endpoint as follows:
+
+    - For an endpoint without parameters, you can click **Send** to call it directly.
+    - For an endpoint with parameters, you need to fill in the values of the parameters first, and then click **Send**.
+
+        - For a `GET` or `DELETE` request, fill in values of the parameters in the **Query Params** table.
+        - For a `POST` or `PUT` request, click the **Body** tab, and then fill in values of the parameters as an JSON object. If **Batch Operation** is enabled for the endpoint in TiDB Cloud Data Service, fill in values of the parameters as an array of JSON objects.
+
+3. Check the response in the lower pane.
+
+4. If you want to call the endpoint again with different parameter values, you can edit the parameter values accordingly, and then click **Send** again.
+
+## Deal with new changes in Data App
+
+After a Data App is imported to Postman, TiDB Cloud Data Service will not synchronize the following new changes to the Data App to Postman automatically.
+
+- The creation of new endpoints
+- The renaming or deletion of existing endpoints
+- The parameter modification of existing endpoints
+
+If you want to have these changes in Postman, you need to import the latest Data App to Postman again. Because the connection name is unique in a Postman workspace, you can either use the latest Data App to replace the previously imported one or import the latest Data App as a new connection.
```

Comment:
```suggestion
If you want to have these changes in Postman, you need to [import the latest Data App](/tidb-cloud/data-service-postman-integration.md#step-1-connect-your-data-app-to-postman) to Postman again. Because the connection name is unique in a Postman workspace, you can either use the latest Data App to replace the previously imported one or import the latest Data App as a new connection.
```